synapse/storage/databases/main/events_worker.py

```diff
@@ -1457,7 +1457,7 @@ def _fetch_event_rows(
                 event_dict[event_id] = _EventRow(
                     event_id=event_id,
                     stream_ordering=row[1],
-                    instance_name=row[2],
+                    instance_name=row[2] or "master",
```

Comment:
Add comment to align with other places we do this:

```suggestion
                    # If instance_name is null we default to "master"
                    instance_name=row[2] or "master",
```